### ISSUE TITLE
Substitution failed at begin of import path

### ISSUE THREAD

**Issue by HolQue** [test-fullautomation/python-jsonpreprocessor#372]
JSON:

```
"current_dir" : ".",
"[import]"    : "${current_dir}/imports/AA/imported.AA.jsonp"
```

Result:

`Error: 'File '%current_dir%__ConvertParameterToString__/imports/AA/imported.AA.jsonp' is not existing!'!`

Expected:

`DotDict({'current_dir': '.', 'AA': 1})`

Similar:

```
"P1" : "./imp",
"P2" : "orts/A",
"P3" : "A/imported.A",
"P4" : "A.jsonp",
"[import]" :  "${P1}${P2}${P3}${P4}"
```

Result:

`Error: 'File '%P1%orts/AA/imported.AA.jsonp' is not existing!'!`

The first substitution fails.

**Comment by HolQue** [comment]
JPP_1104
JPP_1105

**Comment by namsonx** [comment]
Hello Holger,

This issue was solved on stabi branch, could you please help me verify it again?

Thank you,
Son

**Comment by HolQue** [comment]
Retest successful. Issue can be closed.

**Comment by test-fullautomation** [comment]
merged for version 0.13.1
